src/hub.js

```diff
@@ -29,6 +70,31 @@ export function hub(chrome) {
 	 */
 	const appPongs = new Map();
 
+	const appSpawns = [];
+
+	const add = (method, handler) => {
+		methodHandlers.set(method, handler);
+	};
+	const call = (method, argsArr) => {
+		if (methodHandlers.has(method)) {
+			return methodHandlers.get(method).apply({}, argsArr);
+		}
+	};
+
+	add('kill', function(targetWindow) {
```

Comment:
this is the part where we stop pinging the `app` from the `hub` and drop all references to it.

If this code doesn't exist, it will take up to 10 seconds until the `app` is considered dead.

The Chrome still takes care of removing frames, and it also does `hub.call('kill', window)` before doing so.

Reply:
If it's already just removing connections/references, then this should just be a function, it's weird that the hub has a hidden RPC method for it. It should be `hub.remove(someTarget/App)`